modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/display/FeatureFlagsDisplayContextFactory.java

```diff
@@ -164,12 +172,54 @@ private boolean _contains(Locale locale, String s1, String s2) {
 		return normalized.contains(_normalize(locale, s2));
 	}
 
+	private List<FeatureFlag> _getDependentFeatureFlags(
+		long companyId, FeatureFlag featureFlag,
+		FeatureFlagsBag featureFlagsBag) {
+
+		List<String> featureFlagKeys = new ArrayList<>(
```

Comment:
I'm going with `dependencyKeys` and `systemScopedDependencyKeys`. Since we're already in the context of a Feature Flag, it sound redundant to add 'featureFlag' as prefix.